src/components/expandableSection/index.tsx

```diff
@@ -38,25 +45,39 @@ export type ExpandableSectionProps = {
  */
 
 function ExpandableSection(props: ExpandableSectionProps) {
-  const {expanded, sectionHeader, onPress, children, top, testID} = props;
+  const {minHeight, expanded, sectionHeader, onPress, children, top, testID} = props;
   const [height, setHeight] = useState(0);
   const animatedHeight = useSharedValue(0);
+  const shouldShowSectionHeader = !minHeight || height > minHeight;
 
   const onLayout = (event: LayoutChangeEvent) => {
-    const onLayoutHeight = event.nativeEvent.layout.height;
+    const layoutHeight = event.nativeEvent.layout.height;
 
-    if (onLayoutHeight > 0 && height !== onLayoutHeight) {
-      setHeight(onLayoutHeight);
+    if (layoutHeight > 0 && height !== layoutHeight) {
+      setHeight(layoutHeight);
     }
   };
 
-  const expandableStyle = useAnimatedStyle(() => {
-    animatedHeight.value = expanded ? withTiming(height) : withTiming(0);
+  const animateHeight = useCallback((shouldAnimate = true) => {
+    const collapsedHeight = Math.min(minHeight ?? 0, height);
+    const toValue = expanded ? height : collapsedHeight;
+    animatedHeight.value = shouldAnimate ? withTiming(toValue) : toValue;
+  },
+  [animatedHeight, expanded, height, minHeight]);
```

Comment:
no need to break the line here

Reply:
This is how prettify is indenting for me